ci/actions-templates/linux-builds-template.yaml

```diff
@@ -169,6 +169,18 @@ jobs: # skip-master skip-pr skip-stable
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DEFAULT_REGION: us-west-1
+      - name: Configure AWS credentials
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::890664054962:role/ci--rust-lang--rustup
+          aws-region: us-east-1
+      - name: Deploy build to rustup-builds bucket for release team
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
```

Comment:
Are you sure this is `master`? Currently our release builds (beta or not) are based on the `stable` branch.

Reply:
I was certain that in one of the many discussion around this topic it was suggested to upload every push on `master`. But as much as I'm looking for it, I can't find it anymore. So maybe I am mixing up something.

@Mark-Simulacrum Any preferences from the release team's perspective on whether to upload from `master` or `stable`?

Reply:
I'd prefer that we match what we do for rustc, and upload every commit on master too. We probably still need to upload from stable so that backports etc work.

Reply:
I'm not sure how backports in Rustup work, but the dev guide suggests fast-forward merges for regular releases. So both `master` and `stable` should point to the same commit and imo uploading only from `master` should be enough. But I'm happy to be corrected if my understanding is wrong...

Reply:
@jdno Yeah, since rustup is a not so active repo (compared to rust) we currently don't do backports at all.